We have an issue with dlclose on uclibc on x86_64.
For now we work around it by simply not dlcose our plugins.


--- ./src/modules/bank.c.orig
+++ ./src/modules/bank.c
@@ -484,11 +484,13 @@
     assert (module->next == NULL);
 
     /* Unload plugin until we really need it */
+#if 0
     if (module->b_loaded && module->b_unloadable)
     {
         module_Unload (module->handle);
         module->b_loaded = false;
     }
+#endif
 
     /* For now we force loading if the module's config contains
      * callbacks or actions.
